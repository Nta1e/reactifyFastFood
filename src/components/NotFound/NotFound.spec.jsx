import React from 'react';
import { shallow } from 'enzyme';
import NotFound from './NotFound'

describe('test not found',()=>{
    let wrapper;
    beforeEach(()=>{
        wrapper = shallow(<NotFound/>)
    });
    it('matches snapshot', ()=>{
        expect(wrapper).toMatchSnapshot()
    })
})