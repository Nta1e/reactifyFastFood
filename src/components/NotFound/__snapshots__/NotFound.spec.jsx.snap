// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test not found matches snapshot 1`] = `ShallowWrapper {}`;
